### DISCUSSION TITLE
Vendor Payment Reports - API Endpoints

### DISCUSSION THREAD

**Discussion by jilliansd** [amzn/selling-partner-api-models#589] [Ideas] [answered]
**If you need help troubleshooting a problem with Selling Partner APIs, please [open a support case]

Need SP-API help related to Vendor information, the vendor payments information is available for manual report download via the vendor central but I am unable to find any suitable endpoints to obtain vendor payments information for invoices and remittances.

1/ Vendor Central -> payments-> Invoices -> View and download Invoices Information Manually -> how to get this same information via SP-API? Which endpoints are available for payments invoices information related to vendors?

2/ Vendor Central -> payments-> Remittances -> View and download Invoices Information Manually -> how to get this same information via SP-API? Which endpoints are available for payments remittance information related to vendors?

Please share the relevant documentation/endpoint name that provides the vendors payments information

**Describe the improvement**
Please add details on how the following 2 vendor payments related reports can be accessed via the api:

1/ Vendor Central -> payments-> Invoices -> View and download Invoices Information Manually -> how to get this same information via SP-API? Which endpoints are available for payments invoices information related to vendors?

2/ Vendor Central -> payments-> Remittances -> View and download Invoices Information Manually -> how to get this same information via SP-API? Which endpoints are available for payments remittance information related to vendors?

The vendor api documentation has no details related to payments reporting for these items.

**Additional context**
The reports are all available manually from vendor central but no supporting endpoints or documentation to trace how we can fetch this data via api.

**Comment by jilliansd** [comment]
I have raised a support ticket also with the support team seeking more information on these:
ID 8306101402

**Comment by puppsupr** [comment, marked as answer]
Hi @jilliansd
Currently we don't have any api's for vendor payments information for invoices and remittances.

Thanks and Regards,
Supriya P
Solutions Architect, Selling Partner API
[SP API Developer Videos](https://www.youtube.com/@amazon-sp-api) | [SP API Blogs](https://developer-docs.amazon.com/sp-api-blog/docs/getting-started)

**Comment by avlly** [comment]
I upvote on the enhancement of the API to get Invoices and Remittance payments

**Reply by zoemikecw** [reply]
Releasing this would be a huge advance of managing payments and invoices with Amazon and make it much easier for many brands & partners to continue to grow their business with Amazon

**Comment by ratnaprasadt** [comment]
Hi Amazon SP-API support,
We need to download [with applied filter criterion] Remittances and Claims’ documents [Dispute details], via SP-API,
Can you please advice further or with any other alternative for Automation.

**Reply by 5cotts** [reply]
Also looking to download dispute details. Would be very useful if this feature existed programatically.

**Comment by henriquerangel** [comment]
This information is extremely important, and it would be crucial for us to obtain it through an API. Does anyone have any information on how we can achieve this?

**Comment by ChaitanyaChakurshe** [comment]
@puppsupr 
Is there any plan in near future to make Invoices and Remittance payments available via SP-API for vendors?
We are waiting for this endpoints.

**Comment by renatofge** [comment]
It would be a huge improvement, for sure, to use SP-API to get Invoices and Remittance payments data. 
Hope this is in the development pipeline @puppsupr
